integration/wc-integration/src/main/java/se/inera/intyg/privatlakarportal/integration/privatepractitioner/services/ValidatePrivatePractitionerResponderImpl.java

```diff
@@ -33,25 +31,11 @@
  */
 public class ValidatePrivatePractitionerResponderImpl implements ValidatePrivatePractitionerResponderInterface {
 
-    @Autowired
-    private se.inera.intyg.privatlakarportal.integration.privatepractitioner.services.IntegrationService integrationService;
-
     @Override
     public ValidatePrivatePractitionerResponseType validatePrivatePractitioner(String s,
         ValidatePrivatePractitionerType validatePrivatePractitionerType) {
 
-        final boolean hasHsaArgument = !StringUtils.isEmpty(validatePrivatePractitionerType.getPersonHsaId());
-        final boolean hasPersonArgument = !StringUtils.isEmpty(validatePrivatePractitionerType.getPersonalIdentityNumber());
-
-        if (hasHsaArgument && hasPersonArgument) {
-            throw new IllegalArgumentException("Endast ett av argumenten hsaIdentityNumber och personalIdentityNumber får vara satt.");
-        } else if (hasHsaArgument) {
-            return integrationService.validatePrivatePractitionerByHsaId(validatePrivatePractitionerType.getPersonHsaId());
-        } else if (hasPersonArgument) {
-            return integrationService.validatePrivatePractitionerByPersonId(validatePrivatePractitionerType.getPersonalIdentityNumber());
-        } else {
-            throw new IllegalArgumentException(
-                "Inget av argumenten hsaIdentityNumber och personalIdentityNumber är satt. Ett av dem måste ha ett värde.");
-        }
+        throw new UnsupportedOperationException("Method is deprecated and shall be removed.");
```

Comment:
If it is deprecated it should still work, shouldn't it?

Reply:
Changed error message.